src/components/dev-dot-content/dev-dot-content.module.css

```diff
@@ -91,14 +91,16 @@
   }
 
   & table {
+    background-color: transparent;
     border-collapse: collapse;
+    border-right: 0;
     border-spacing: 0;
     border: 1px solid #ddd;
-    border-right: 0;
-    background-color: transparent;
-    width: 100%;
-    max-width: 100%;
+    display: block;
     margin-bottom: 20px;
+    max-width: 100%;
+    overflow-x: auto; /* TODO: find alternative solution */
```

Comment:
⛏️ nit: I'm not sure what a _better_ solution would be 🤔

Reply:
Mainly something that doesn't involve horizontal scrolling. It makes content completely inaccessible to keyboard-only users.

Reply:
Gotcha, can we specify that in the comment? That would be a helpful breadcrumb!